MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/Views/Home/Index.cshtml

```diff
@@ -15,32 +15,32 @@
         <div id="index-flex">
             
             <div id="index-previews">
-                <a href="../Home/Gallery">
+                <a class="linksToPages" href="../Home/Gallery">
```

Comment:
You can use `href="@Url.Action("Gallery", "Home")"`  here (and similar on the other links) - which will do the same thing but means that if we changed our urls it would update them automatically (similar to how @html.ActionLink works)